tests/serial/initialize-data/ImplicitBoth.xml

```diff
@@ -42,8 +42,8 @@
     <participants first="SolverOne" second="SolverTwo" />
     <max-time-windows value="10" />
     <time-window-size value="1.0" />
+    <min-iterations value="3" />
```

Comment:
```suggestion
    <min-iterations value="5" />
```

Reply:
What happened in this test before? Did we use min or max iterations? Maybe also something we should consider cleaning up on `develop` first before touching it here.

Reply:
Tests implicit coupling with initializeData on both exchanges. Iterations don't seem to matter here.